pkg/distributor/distributor.go

```diff
@@ -141,6 +154,11 @@ func New(cfg Config, clientConfig ingester_client.Config, limits *validation.Ove
 	replicationFactor.Set(float64(ring.ReplicationFactor()))
 	cfg.PoolConfig.RemoteTimeout = cfg.RemoteTimeout
 
+	replicas, err := NewClusterTracker(cfg.TrackerConfig)
```

Comment:
Probably only want to instantiate this is cfg.HAReplicas is true?  Won't it fail if consul flags aren't specified?

Reply:
Did you want to keep this config flag in addition to the per user "accept ha samples" flag